### PR TITLE
Switch from symlink to PATH in example

### PR DESCRIPTION
**Pull request by ddelange**
Hi 👋 

Let me know what you think :)

```console
$ which python
/usr/share/python3/app/bin/python
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,8 +114,8 @@ COPY --from=builder /usr/share/python3/app /usr/share/python3/app
 # Install the required library packages
 RUN xargs -ra /usr/share/python3/app/pkgdeps.txt apt-install
 
-# Create a symlink to the target binary (just for convenience)
-RUN ln -snf /usr/share/python3/app/bin/ipython /usr/bin/
+# Add entrypoints from the venv to PATH (e.g. 'pip', 'python', just for convenience)
+ENV PATH="/usr/share/python3/app/bin:${PATH}"
```
Comment (mosquito) [inline README.md]: Yes it's good but in this case the `python3` starts running from `/usr/share/python3/app/bin` it's not safe default IMHO.

Might be add a commented line with this alternative way and describe it.

**Comment by mosquito** [comment]
@dudelange thank you for a question, really like this format the PR instead of issues ;-)

I answered in the code comments what am I think about it.

File: README.md (hunk 1 above)
Comment (ddelange) [reply README.md]:
```suggestion
ENV PATH="${PATH}:/usr/share/python3/app/bin"
```

Is it safer like this?

Comment (ddelange) [reply README.md]:
```console
$ which python
/usr/share/python3/app/bin/python
$ which python3
/usr/bin/python3
```

Comment (mosquito) [reply README.md]: Inconsistent with python2 behaviour

**Review by mosquito — APPROVED** [review]

**Comment by ddelange** [comment]
Hmm, CI fails due to missing credentials 🤔 

> Inconsistent with python2 behaviour

Agreed, is it a problem? Should we note it in that comment and go ahead with this PR? Or close it and keep the softlink?

File: README.md [hunk 2]
```diff
@@ -114,8 +114,8 @@ COPY --from=builder /usr/share/python3/app /usr/share/python3/app
 # Install the required library packages
 RUN xargs -ra /usr/share/python3/app/pkgdeps.txt apt-install
 
-# Create a symlink to the target binary (just for convenience)
-RUN ln -snf /usr/share/python3/app/bin/ipython /usr/bin/
+# Add entrypoints from the venv to PATH (e.g. 'pip', 'python', just for convenience)
```
Comment (ddelange) [inline README.md:117]:
```suggestion
# For convenience, add entrypoints from the venv to PATH, e.g. 'pip', 'python' (on py2 /usr/bin/python comes first)
```

**Comment by ddelange** [comment]
hmm, the above changes won't work. both the `push` and `pull` are needed (always building upon `latest`). the alternative isn't very clean... https://github.com/docker/build-push-action/issues/225#issuecomment-727639184

**Comment by ddelange** [comment]
Solution would probably be to pass the default available gh token to the docker login action for ghcr ([docs](https://github.com/docker/login-action#github-container-registry)): `password: ${{ secrets.GITHUB_TOKEN }}` but that doesn't solve the issue of pulling `latest` regardless of branch...

Would say it's a separate project to get CI to run for PRs, and due to the inconsistency with python2, will close this PR.

**Pull request closed without merging**